task/04-arrays-tasks.js

```diff
@@ -286,7 +327,10 @@ function propagateItemsByPositionIndex(arr) {
  *   [ 10, 10, 10, 10 ] => [ 10, 10, 10 ]
  */
 function get3TopItems(arr) {
-   throw new Error('Not implemented');
+   const arrOf3TopItems = arr.sort((a, b) => {
```

Comment:
Never change arguments. It is a potential bug. Why? `sort` operator change `arr` and external code will work with mutated array.